airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -2139,3 +2139,11 @@
   documentationUrl: https://docs.airbyte.io/integrations/sources/braze
   sourceType: api
   releaseStage: alpha
+- name: Databend
+  sourceDefinitionId: d2afb200-c46f-4087-a802-faac426b581c
+  dockerRepository: airbyte/source-databend
+  dockerImageTag: 0.1.0
+  documentationUrl: https://docs.airbyte.io/integrations/sources/databend
+  sourceType: database
+  icon: databend.svg
```

Comment:
please can you add the databend icon to the folder or remove this?

Reply:
The databend.svg already exist in the folder because the `databend-destination` has been merged.